src/FffGcodeWriter.cpp

```diff
@@ -59,16 +59,15 @@ void FffGcodeWriter::writeGCode(SliceDataStorage& storage, TimeKeeper& time_keep
         processNextMeshGroupCode(storage);
     }
 
+    bool doing_spiralize = false;
     size_t total_layers = 0;
     for (SliceMeshStorage& mesh : storage.meshes)
     {
         total_layers = std::max(total_layers, mesh.layers.size());
-        if (mesh.getSettingBoolean("magic_spiralize"))
-        {
-            findLayerSeamsForSpiralize(mesh);
-        }
 
         setInfillAndSkinAngles(mesh);
+
+        doing_spiralize = doing_spiralize || mesh.getSettingBoolean("magic_spiralize");
```

Comment:
I thin kthat with the new spiralize implementation it doesn't really make sense any more to be able to set spiralize per object.
You can just do `storage.getSettingBoolean("magic_spiralize")` instead of looping over all meshes.

fdmprinter.def.json should then get
```
                    "settable_per_mesh": false,
                    "settable_per_extruder": false
```